From 4f1be619107b82b43fe58aaf116a1cae42a01bb5 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 27 Jul 2018 17:51:08 +0800
Subject: [PATCH 4397/5242] MLK-19067-2: ARM64: dts: only support playback on
 imx8mm evk

commit  d36de470c5049afcf2fb6ff13d108fa7c611ae82 from
https://source.codeaurora.org/external/imx/linux-imx.git

only support playback for rpmsg device on imx8mm evk.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit a6e9ff5efc814fa08f7e4664f584861b4a4d5792)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
index 3e68451..1f9c667 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
@@ -47,6 +47,7 @@
 		compatible = "fsl,imx-audio-rpmsg";
 		model = "rpmsg-audio";
 		cpu-dai = <&rpmsg_i2s>;
+		rpmsg-out;
 	};
 
 };
-- 
1.7.9.5

